addon/utils/get-code.js

```diff
@@ -11,17 +11,22 @@ export default function getCode(event) {
 
   if (!code.includes('Key') || !key) {
     return codeMap[keyCode] || code;
-  }
+	}
+	
+	// Handle software map keyboards that do include KeyX
```

Comment:
Can you provide an example in the comment of what's expected here and why it's different?

Reply:
Yea, I'll update this on Monday. Basically on the latest version of this addon, when I use (on my mac) alt + c, it actually registers as alt + ç. Which doesn't fire when I'm trying to fire on alt + c. I think it may because I'm using Karabiner Elements for soft key-remap, but I'll verify that as well. Either way, this update fixes the issue since it's not handled.

Reply:
So, what is
- the physical key you're pressing?
- the key that's printed, like in the console or whatever
- what key do you expect to see?

I have concerns over this approach, and I,m not quite sure it's correct, or 'accidentally works', ya know?

Reply:
- alt + c key are the physical keys. 
- The key that is printed is ç.
- I expect to see alt+c
- The problem here is that the key-remapping section ignores the `Code` and tries to create it's own via the letter. But on mac's alt + c sends ç not c. So there is a re-map between step where the re-mapping isn't for DVORAK, but someone like me who uses a windows keyboard and just re-mapped the meta keys.

Reply:
Interesting. I wonder if we should have a tests file solely for explaining all this mapping silliness

Reply:
FYI, the alt+letter behavior described by @optikalefx is reproducible on my mac without any unusual keyboard or key mapping.